src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/avsluttbehandling/AvsluttBehandlingStegTest.kt

```diff
@@ -0,0 +1,135 @@
+package no.nav.familie.ks.sak.kjerne.behandling.steg.avsluttbehandling
+
+import io.mockk.every
+import io.mockk.impl.annotations.InjectMockKs
+import io.mockk.impl.annotations.MockK
+import io.mockk.junit5.MockKExtension
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.runs
+import io.mockk.slot
+import io.mockk.verify
+import no.nav.familie.ks.sak.common.exception.Feil
+import no.nav.familie.ks.sak.data.lagAndelTilkjentYtelse
+import no.nav.familie.ks.sak.data.lagBehandling
+import no.nav.familie.ks.sak.data.lagInitieltTilkjentYtelse
+import no.nav.familie.ks.sak.kjerne.behandling.BehandlingService
+import no.nav.familie.ks.sak.kjerne.behandling.domene.Behandling
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingStatus
+import no.nav.familie.ks.sak.kjerne.behandling.domene.Behandlingsresultat
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
+import no.nav.familie.ks.sak.kjerne.beregning.BeregningService
+import no.nav.familie.ks.sak.kjerne.fagsak.FagsakService
+import no.nav.familie.ks.sak.kjerne.fagsak.domene.FagsakStatus
+import no.nav.familie.ks.sak.kjerne.logg.LoggService
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import java.time.YearMonth
+
+@ExtendWith(MockKExtension::class)
+internal class AvsluttBehandlingStegTest {
+
+    @MockK
+    private lateinit var behandlingService: BehandlingService
+
+    @MockK
+    private lateinit var loggService: LoggService
+
+    @MockK
+    private lateinit var beregningService: BeregningService
+
+    @MockK
+    private lateinit var fagsakService: FagsakService
+
+    @InjectMockKs
+    private lateinit var avsluttBehandlingSteg: AvsluttBehandlingSteg
+
+    private lateinit var behandling: Behandling
+
+    @BeforeEach
+    fun init() {
+        behandling = lagBehandling(opprettetÅrsak = BehandlingÅrsak.SØKNAD)
+            .also { it.status = BehandlingStatus.IVERKSETTER_VEDTAK }
+
+        every { behandlingService.hentBehandling(behandling.id) } returns behandling
+        every { loggService.opprettFerdigstillBehandling(behandling) } just runs
+    }
+
+    @Test
+    fun `skal ikke utføre steg når behandling ikke har IVERKSETT_VEDTAK status`() {
```

Comment:
Bare for å fortsette med vår navn-pattern på tester, så er det sikkert greit å prefixe testen med metodenavnet (`utførSteg`) her.